spot-ml/src/main/scala/org/apache/spot/dns/DNSWordCreation.scala

```diff
@@ -90,8 +79,8 @@ class DNSWordCreation(frameLengthCuts: Array[Double],
     * @return The word representation of the DNS entry.
     */
 
-  def dnsWord(timeStamp: String,
-              unixTimeStamp: Long,
+  def dnsWord(timestamp: String,
```

Comment:
the English word is "time stamp" so timeStamp is the correct rendering in camel case
http://www.dictionary.com/browse/time--stamp?s=t